dev-docs/bidders/turktelekom.md

```diff
@@ -1,19 +1,27 @@
 ---
 layout: bidder
 title: Türk Telekom
-description: Prebid Türk Telekom Bidder Adaptor
+description: Türk Telekom Bidder Adaptor
 pbjs: true
+pbs: true
 biddercode: turktelekom
-media_types: banner, video
+aliasCode : adkernel
+media_types: banner, native, video
 gdpr_supported: true
-pbjs_version_notes: not in 5.x
+usp_supported: true
+coppa_supported: true
+pbs_app_supported: true
+schain_supported: true
```

Comment:
When you're at it. Could you add the information if safeframes work?

```
safeframes_ok: true
```

means your adapter works with safeframes, otherwise it's `false`